src/components/Elevation/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
+import useStyles from './styles';
+
+export interface IElevationProps {
+  variant?: 'extraLight' | 'light' | 'medium' | 'heavy' | 'extraHeavy';
+  children: React.ReactNode;
+  style?: StyleProp<ViewStyle>;
+}
+
+const Elevation: React.FC<IElevationProps> = ({
+  variant = 'medium',
+  children,
+  style,
+}) => {
+  const styles = useStyles();
+
+  return (
+    <View style={[styles.wrapper, styles[variant], style]}>{children}</View>
```

Comment:
I think it should have width and height props. Because additional styles should be the last option.

Reply:
I think we should add props for all these styles like margin, padding, width, height, justify, align, radius

margin: 20,
      width: 216,
      height: 216,
      justifyContent: 'center',
      alignItems: 'center',
      borderRadius: 10,